examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -268,7 +273,7 @@ export const ExternalServerTaskListView: React.FC<ExternalServerTaskListViewProp
 					"Access-Control-Allow-Origin": "*",
 					"Content-Type": "application/json",
 				},
-				body: JSON.stringify({ taskList: formattedTasks }),
+				body: JSON.stringify({ taskList: { [taskListId]: formattedTasks } }),
```

Comment:
Not a huge deal but this feels like it would be cleaner in the URL, either `.../set-tasks/{taskListId}` or `.../set-tasks?taskListId={taskListId}`.